vra/resource_load_balancer.go

```diff
@@ -152,7 +131,7 @@ func resourceLoadBalancerCreate(d *schema.ResourceData, m interface{}) error {
 		Pending:    []string{models.RequestTrackerStatusINPROGRESS},
 		Refresh:    loadBalancerStateRefreshFunc(*apiClient, *createLoadBalancerCreated.Payload.ID),
 		Target:     []string{models.RequestTrackerStatusFINISHED},
-		Timeout:    d.Timeout(schema.TimeoutCreate),
+		Timeout:    5 * time.Minute,
```

Comment:
Revert this change as it removes the capability to specify custom timeouts for provisioning.